contentcuration/contentcuration/utils/publish.py

```diff
@@ -437,6 +437,10 @@ def create_perseus_zip(ccnode, exercise_data, write_to_path):
                     write_assessment_item(question, zf)
                 except Exception as e:
                     logging.error("Publishing error: {}".format(str(e)))
+                    # For now, raise errors during exercise publishing so we can determine specific errors that
```

Comment:
We will want to avoid this later as it could potentially block some rather large channels from publishing